libr/anal/p/anal_x86_cs.c

```diff
@@ -3596,6 +3596,7 @@ static RList *anal_preludes(RAnal *anal) {
 	case 64:
 		KW ("\x55\x48\x89\xe5", 4, NULL, 0);
 		KW ("\x55\x48\x8b\xec", 4, NULL, 0);
+		KW ("\xf3\x0f\x1e\xfa", 4, NULL, 0); // endbr64
```

Comment:
since you are at it, could you also add `endbr32` for the 32 bits case? `endbr32` is `F30F1EFB`